tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-loader-test.php

```
@@ -0,0 +1,361 @@
+					\Automattic\WooCommerce\Enums\ProductType::SIMPLE,
+					\Automattic\WooCommerce\Enums\ProductType::VARIATION,
+					\Automattic\WooCommerce\Enums\ProductType::VARIABLE,
+				],
+				'limit'  => -1,
+				'return' => 'ids',
+			]
+		);
+		foreach ( $existing as $product_id ) {
+			WC_Helper_Product::delete_product( $product_id );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method set_up_before_class is not named in camelCase. (undefined)

(CamelCaseMethodName)